app/editor/src/components/wysiwyg/Wysiwyg.tsx

```diff
@@ -100,20 +104,41 @@ export const Wysiwyg = <T extends object>({
           .replace(/<br\s*\/?>/g, '[br]'),
         'text/html',
       );
-      doc.body.textContent =
+      const html =
         doc.body.textContent
           ?.replaceAll('[p]', '<p>')
           .replaceAll('[/p]', '</p>')
           .replaceAll('[br]', '<br>') || '';
-      setFieldValue(fieldName as string, doc.body.textContent);
-      setState({ ...state, html: doc.body.textContent });
+      if (html !== state.html) {
+        setState({ ...state, html: html });
+        setFieldValue(fieldName as string, html);
+      }
     }
   };
 
+  const extractTags = (values: string[]) => {
+    return tags
+      .filter((tag) =>
+        values.some((value: string) => value.trim().toLowerCase() === tag.code.toLowerCase()),
+      )
+      .map((tag) => tag);
```

Comment:
Why do you use `map()` here?

Reply:
Good catch, updated as this was the previous code for handling the similar situation, thanks